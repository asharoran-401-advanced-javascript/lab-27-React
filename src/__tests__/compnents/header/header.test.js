// eslint-disable-next-line strict
'use strict';

import React from 'react';
import { shallow , mount} from 'enzyme';
import renderer from 'react-test-renderer';
import Header from '../../../components/header/header.js';

describe(' <Header />' , () =>{
    it('header is exist at start application' , () =>{
        let app = shallow(<Header />);
        expect(app.find('h1').exists).toBeTruthy();
    });
    it('render correctly',()=>{
        const tree = renderer.create(<Header />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});


