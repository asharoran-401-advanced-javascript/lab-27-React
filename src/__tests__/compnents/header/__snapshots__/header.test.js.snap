// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` <Header /> render correctly 1`] = `
<header>
  <h1>
    Counter for React Application
  </h1>
</header>
`;
